proxyservice/jenkins.go

```diff
@@ -116,3 +116,25 @@ func (j *Jenkins) TriggerDockerhubJob(data *DockerHubWebhookData) error {
 	params.Set("RawJSON", string(rawJSON))
 	return j.TriggerJob(path, params)
 }
+
+// triggers a jenkins job given
+func (j *Jenkins) TriggerTaskclusterJob(taskID string, route string, data *TaskCompletedMessage) error {
```

Comment:
Can you document what route will look like?

Reply:
I've added my initial thoughts on what this would look like [here](https://github.com/mozilla-releng/shipit/pull/378#issue-465782039).